src/constants/api-constants.ts

```diff
@@ -0,0 +1,9 @@
+const API_URLS = Object.freeze({
+  API_CRI_CALLBACK: "/cri/callback",
+  API_MOBILE_APP_CALLBACK: "/app/callback",
+  API_JOURNEY_EVENT: "/journey",
+  API_SESSION_INITIALISE: "/session/initialise",
+  API_BUILD_PROVEN_USER_IDENTITY_DETAILS: "/user/proven-identity-details",
+});
+
```

Comment:
could we pull these directly from config.ts rather than having to redefine them here?